src/lib/HTTPError.ts

```diff
@@ -1,8 +1,10 @@
+type Body = { [key: string]: any }
+
 export class HTTPError extends Error {
   public statusCode: number
-  public body?: string
+  public body?: Body | string
 
-  constructor(message: string, statusCode: number, body?: string) {
+  constructor(message: string, statusCode: number, body?: Body | string) {
```

Comment:
When testing, `body` was not a string, but already JSON-parsed (possibly by the `request` lib we use)